### PR TITLE
Refactor healing feedback

### LINKED ISSUE #1530: [BUG] The healing effect is shown at the beginning of the match
## 🐛 Bug Report

The healing effect starts at the beginning of the match

### Steps to Reproduce

Enter muflus prefab or start a match to see this

### Posible solutions

The healing feedback's child starts active. Maybe deactivating it and activating it when it's needed solves this
Captura de pantalla 2024-02-28 a la(s) 15 10 27

### PR DESCRIPTION
**Pull request by AngieDutra**
Closes #1530 

## Motivation

The healing feedback was being trigger when a match started. This is not ideal behavior.

## Summary of changes

The healing feedback was a parent to the vfx since before it's children were particle systems. Now we have only one vfx healing feedback so I deleted the vfx child and added the vfx to the HealFeedback game object.
In addition, the vfx was set to play on enable. This resulted on it showing at the beginning with the character load in the scene. I set the vfx to be played only by code.
Captura de pantalla 2024-03-04 a la(s) 13 42 55

## How has this been tested?

- Start a game and check if the healing feedback is working as expected. 
- To further check this, open any character prefab with the scene opened too. The feedback won't show. If this were to be done in main the healing feedback would show
Captura de pantalla 2024-03-04 a la(s) 13 41 13

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Review by BertovDev — APPROVED** [review]

**Review by Nico-Sanchez — CHANGES_REQUESTED** [review]
The healing feedback animation is still being triggered for Muflus. Tested with all the characters, but only Muflus is still showing it when he spawns.

File: client/Assets/Scripts/CustomLevelManager.cs [hunk 1]
```diff
@@ -84,9 +84,12 @@ private IEnumerator InitializeLevel()
         playerToFollowId = playerId;
         GeneratePlayers();
         SetPlayersSkills(playerId);
-        var player = Utils.GetPlayer(playerId);
         SetOrientationArrow(playerId);
 
+        Entity gamePlayer = Utils.GetGamePlayer(playerId);
+        GameObject player = Utils.GetPlayer(playerId);
+        player.GetComponent<Health>().CurrentHealth = gamePlayer.Player.Health;
+
```
Comment (BertovDev) [inline client/Assets/Scripts/CustomLevelManager.cs:92]: You can simplify this in `GeneratePlayers` method with this:

`newPlayer.CharacterHealth.CurrentHealth = player.Player.Health;`

You can write this above the line `173` in `CustomLevelManager`

**Review by BertovDev — CHANGES_REQUESTED** [review]
Niceee, just a small change and we are done!

File: client/Assets/Scripts/CustomLevelManager.cs (hunk 1 above)
Comment (AngieDutra) [reply client/Assets/Scripts/CustomLevelManager.cs:92]: Done!

**Review by BertovDev — APPROVED** [review]

**Review by Nico-Sanchez — APPROVED** [review]

**Pull request merged**